CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # CHANGE LOG
 
+## 3.15.1
+
+### Features
+- Algolia's [Query Categorization feature](https://www.algolia.com/doc/integration/magento-2/how-it-works/query-categorization) is now compatible with the extension.
+
+### Updates
+- Updated integration tests to use `AlgoliaConnector`
+- Updated `ReplicaManager` service to handle multi stores properly with a `$storeId` parameter.
```

Comment:
I'm thinking we should make mention of the PHP API client getting pinned to 4.18.3.